cloudevents/working-drafts/xml-format.md

```diff
@@ -0,0 +1,277 @@
+# XML Event Format for CloudEvents
+
+## Abstract
+
+This format specification for CloudEvents defines how events are expressed
+in XML documents.
+
+## Status of this document
+
+This document is a working draft.
+
+## Table of Contents
+
+1. [Introduction](#1-introduction)
+2. [Attributes](#2-attributes)
+3. [Data](#3-data)
+4. [Envelope](#4-envelope)
+5. [XML Batch Format](#5-xml-batch-format)
+6. [Examples](#6-examples)
+
+## 1. Introduction
+
+[CloudEvents][ce-spec] is a standardized and protocol-agnostic definition of the
+structure and metadata description of events. This specification defines how
+elements defined in the CloudEvents specification are to be represented using
+[Extensible Markup Language (XML)](xml-spec) documents.
+
+* The [Attributes](#2-attributes) section describes the representation and
+data type mappings for CloudEvents context attributes.
+
+* The [Data](#3-data) section defines the container for the data portion of a
+CloudEvent.
+
+* The [Envelope](#4-envelope) section defines an XML element for CloudEvents
+attributes and an associated media type.
+
+* The [Batch](#5-xml-batch-format) section describes how multiple CloudEvents
+can be packaged into a single XML document.
+
+### 1.1. Conformance
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC2119][rfc2119].
+
+### 1.2 Approach
+
+This XML representation used is deliberately verbose in favor of readability
+and deterministic processing.
+
+Preservation of type information for CloudEvent attributes is supported allowing
+custom extensions to be communicated without type loss.
+
+A schema-less approach has been taken favoring convention over rigid document
+structure.
+
+The namespace `http://cloudevents.io/xmlformat/v1` SHOULD be used, when
```

Comment:
Three points:

1) This namespace isn't quite the same as the one used everywhere else - it's got "v1" where all the examples have "V1". I don't know which we should use, but we should make them consistent.
2) I'm interested in why this is only a SHOULD. That feels like anything processing XML representations has to deal with namespaced and non-namespaced elements, which is painful. Could we make this "MUST"?
3) I'd put the namespace prefixes part into a new sentence.

Reply:
Makes sense ... now that namespaces are back it definitely makes sense to move to a MUST.

Will address the other points as well.